### PR TITLE
[Localization] Splash texts Updates

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
Translated new splash texts

## Why am I making these changes?
To have new splash texts translated 

## What are the changes from a developer perspective?
None

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/de/splash-messages.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} Kämpfe gewonnen!",
+  "battlesWon": "Kämpfe gewonnen!",
   "joinTheDiscord": "Tritt dem Discord bei!",
   "infiniteLevels": "Unendliche Level!",
-  "everythingStacks": "Alles stapelt sich!",
+  "everythingIsStackable": "Everything Is Stackable*!",
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:5]:
```suggestion
  "everythingIsStackable": "Alles ist stapelbar*!",
```

File: src/locales/de/splash-messages.json [hunk 2]
```diff
@@ -14,9 +14,9 @@
   "infiniteFusionAtHome": "Wir haben Infinite Fusionen zu Hause!",
   "brokenEggMoves": "Übermächtige Ei-Attacken!",
   "magnificent": "Herrlich!",
-  "mubstitute": "Melegator!",
+  "doPeopleReadThis": "Do People Read This?",
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:17]:
```suggestion
  "doPeopleReadThis": "Liest das jemand?",
```

File: src/locales/de/splash-messages.json [hunk 3]
```diff
@@ -14,9 +14,9 @@
...
   "thatsCrazy": "Das ist verrückt!",
-  "oranceJuice": "Orangensaft!",
+  "gottaCatchEmAll": "Gotta Catch 'Em All!",
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:19]:
```suggestion
  "gottaCatchEmAll": "Komm' und schnapp sie dir!",
```

File: src/locales/de/splash-messages.json [hunk 4]
```diff
@@ -25,12 +25,120 @@
   "moreAddictiveThanIntended": "Süchtig machender als beabsichtigt!",
   "mostlyConsistentSeeds": "Meistens konsistente Seeds!",
   "achievementPointsDontDoAnything": "Errungenschaftspunkte tun nichts!",
-  "youDoNotStartAtLevel": "Du startest nicht auf Level 2000!",
-  "dontTalkAboutTheManaphyEggIncident": "Wir reden nicht über den Manaphy-Ei-Vorfall!",
+  "nothingBeatsAJellyFilledDonut": "Nothing Beats a Jelly-Filled Donut!",
+  "dontTalkAboutTheTinkatonIncident": "Don't Talk About the Tinkaton Incident!",
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:29]:
```suggestion
  "nothingBeatsAJellyFilledDonut": "Donuts mit Marmelade, es gibt nicht besseres auf der Welt!",
  "dontTalkAboutTheTinkatonIncident": "Wir reden nicht über den Granforgita-Vorfallt!",
```

The Donut one is hard. The german version of this definitly isnt that memeable. 
But Rocko (Brock) says "es gibt nicht besseres auf der Welt" to them so idk

File: src/locales/de/splash-messages.json [hunk 5]
```diff
@@ -25,12 +25,120 @@
...
   "alsoTryPokengine": "Versuche auch Pokéngine!",
   "alsoTryEmeraldRogue": "Versuche auch Emerald Rogue!",
   "alsoTryRadicalRed": "Versuche auch Radical Red!",
   "eeveeExpo": "Evoli-Expo!",
-  "ynoproject": "YNO-Projekt!",
-  "breedersInSpace": "Züchter im Weltall!"
-}
\ No newline at end of file
+  "checkOutYnoproject": "Check out YNOproject!",
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:34]:
```suggestion
  "checkOutYnoproject": "Schau dir YNOproject an!",
```

File: src/locales/de/splash-messages.json [hunk 6]
```diff
@@ -25,12 +25,120 @@
...
+  },
+  "xmas": {
+    "happyHolidays": "Happy Holidays!",
+    "delibirdSeason": "Delibird Season!",
+    "unaffilicatedWithDelibirdServices": "Unaffiliated With Delibird Services!",
+    "diamondsFromTheSky": "Diamonds From the Sky!",
+    "holidayStylePikachuNotIncluded": "Holiday Style Pikachu Not Included!"
+  },
+  "newYears": {
+    "happyNewYear": "Happy New Year!"
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:142]:
```suggestion
  "breedersInSpace": "Züchter im Weltall!",
  "alsoTryPokemonUnbound": "Verusch mal Pokemon Unbound!",
  "tryTheJohtoDragonChallenge": "Versuch die Johto Mono-Drache Herausforderung!",
  "basicReadingAbilityRecommended": "Grundlegendes Leseverständnis empfohlen!",
  "shoutoutsToTheArtists": "Shoutout an die Künstler!",
  "gamblingNotEncouraged": "Glücksspiel wird nicht gefördert!",
  "dontForgetToTakeABreak": "Vergiss nicht eine Pause einzulegen!",
  "wEvent": "Geiles Event",
  "ifItsNotAccurateItsAccurate": "Hat es keine 100%ige Genauigkeit trifft es nur in 50% der Fälle.",
  "everyLossIsProgressMade": "Jede Niederlage ist Fortschritt!",
  "liveWoChienReaction": "Live Chongjian Reaktion:",
  "itsAFeatureNotABug": "Es ist ein Feature und kein Fehler!",
  "theEggsAreNotForEating": "Die Eier sind nicht zum Essen da!",
  "7.8outOf10TooManyWaterBiomes": "7.8/10, Zu viel Wasser!",
  "butNothingHappened": "Nichts geschieht!",
  "thePowerOfScienceIsAmazing": "Die Wissenschaft kennt keine Grenzen!",
  "freeToPlay": "Free To Play!",
  "theresATimeAndPlaceForEverything": "Dies ist weder der rechte Ort noch die rechte Zeit dafür",
  "nowWithShinierShinies": "Mit schillerenden schillernden Pokémon!",
  "smilesGoForMiles": "Mit einem Lachen werden wir es machen!",
  "certainlyNotDragonFree": "Sicherlich nicht ohne Drachen!",
  "haveANiceDay": "Hab einen schönen Tag!",
  "redacted": "[ZENSIERT]",
  "hi": "hi",
  "transRights": "Transrechte!",
  "shinyOddsHigherThanYouThink": "Die Chance für schillernde Pokémon ist höher als du denkst!",
  "noFalseTrades": "Keine falschen Pokémon-Täusche!",
  "notForProfit": "Nicht profitorientiert!",
  "timeForYourDailyRun": "Zeit für deinen Täglichen Durchlauf!",
  "moreEggsThanADaycare": "Mehr Eier als der Pokémon-Hort!",
  "disclaimerHarshSunDoesNotGiveVitaminD": "Warnung: Gleißendes Sonnenlicht liefert KEIN Vitamin D!!",
  "whoNeedsAMap": "Wer braucht schon eine Karte?",
  "luxrayIsNotADarkType": "Luxtra ist nicht vom Typ Unlicht!",
  "selfDestructiveEncounters": "Gegner mit Selbstzerstörungstendenzen!",
  "mostOptionsAreViable": "Die meisten Pokémon sind praktikable Optionen!",
  "pokerogueMorse": ".--. --- -.- . .-. --- --. ..- .",
  "smiley": ":)",
  "beAwareOfPassives": "Achte auf passive Fähigkeiten!",
  "asSeenOnTheWorldWideWeb": "Wie Im Internet zu sehen!",
  "vaultinVeluzas": "Mein lieber Scholli!",
  "tooManyStarters": "Zu viele Starter!",
  "checkTheWiki": "Schau ins Wiki!",
  "winWithYourFavorites": "Gewinne mit deinen Favoriten!",
  "alsoTryPokerogueWait": "Probiere auch PokéRogue! Warte mal...",
  "theWayISeeItKyogreIsSurrounded": "So wie ich das sehe ist Kyogre umzingelt...",
  "tryOutHoneyGather": "Probiere es mal mit Honigmaul!",
  "notForTheFaintOfHeart": "Nichts für Zartbesaitete!",
  "p": "(P)",
  "flipYourDeviceToEvolveInkay": "Dreh dein Gerät auf den Kopf um Iscalar zu entwickeln!",
  "inArceusWeTrust": "Wir vertrauen auf Arceus",
  "whyDidTheTorchicCrossTheRoad": "Warum überquerte das Flemmli die Straße?",
  "goodLuck": "Viel Glück!",
  "fuseWisely": "Fusioniere weise!",
  "compensation": "Entschädigung?",
  "prepareForTroubleAndMakeItDouble": "Jetzt gibt es Ärger...und es kommt noch härter!",
  "anEggForYourTroubles": "Ein Ei für deine Mühen?",
  "regirock": "ÜN ÜN ÜN",
  "hereForAGoodTime": "Hier für eine gute Zeit!",
  "getGoodOrDont": "Werd gut! Oder lass es!",
  "checkTheSubreddit": "Schau dir das Subreddit an!",
  "betterNerfGreninja": "Schwäche Quajutsu lieber ab!",
  "inCaseOfUpdateClearYourCache": "Update? Cache leeren!",
  "insertTextHere": "*hier Text einfügen*",
  "endingEndlessNotFound": "ending_endless nicht gefunden",
  "iLikeMyEggsVouchered": "Ich mag meine Eier mit Gutscheinen!",
  "YOU": "DU!",
  "noAddedSugar": "Ohne Zucker!",
  "notSponsored": "Nicht gesponsert!",
  "notRated": "Nicht eingestuft!",
  "justOneMoreWaveMom": "Nur noch eine Welle, Mama!",
  "saltCured": "Gepökelsalzt!",
  "onlyOnPokerogueNet": "Nur auf pokerogue.net!",
  "pixelPerfection": "Pixel Perfektion!",
  "openSource": "Open Source!",
  "probablyGood": "Wahrscheinlich ganz gut!",
  "itsAMonsterHouse": "Ein Monster-Raum!",
  "dontForgetYourPassword": "Vergiss dein Passwort nicht!",
  "tripleTripleTripleAxel": "Dreifach- Dreifach- Dreifach-Axel!",
  "questionExclamation": "?!",
  "clownEncounters": "Clown Begegnungen!",
  "fullOfBerries": "Voll mit Beeren!",
  "limitsAreMeantToBeBrokenSometimes": "Regeln sind dazu da, gebrochen zu werden, manchmal!",
  "keepItCasual": "Lässig bleiben!",
  "serversProbablyWorking": "Server funktionieren wahrscheinlich!",
  "mew": "Mew ist wahrscheinlich nicht unter dem LKW!",
  "makeItRainAndYourProblemsGoAway": "Goldrausch und deine Probleme sind vergessen!",
  "customMusicTracks": "Eigene Musiktitel!",
  "youAreValid": "Du bist wertvoll!",
  "number591IsLookingOff": "Nummer 591 sieht ein bisschen...",
  "timeForYourDeliDelivery": "Zeit für deine Botogel-Lieferung!",
  "goodFirstImpression": "Hoffe wir haben einen guten ersten Eindruck hinterlassen!",
  "iPreferRarerCandies": "Ich bevorzuge Supersonderbonbons!",
  "halloween": {
    "pumpkabooAbout": "Irrbis überall!",
    "mayContainSpiders": "Könnte Spuren von Spinnen enthalten!",
    "spookyScarySkeledirge": "Gespenstisch gruselige Skelokrok!",
    "gourgeistUsedTrickOrTreat": "Pumpdjinn setzt Halloween ein!",
    "letsSnuggleForever": "Herzliche Knuddelkloppe!"
  },
  "xmas": {
    "happyHolidays": "Fröhliche Feiertage!",
    "delibirdSeason": "Botogel-Saison!",
    "unaffilicatedWithDelibirdServices": "Kein Zusammenhang mit Botogel-Diensten!",
    "diamondsFromTheSky": "Diamantregen vom Himmel!",
    "holidayStylePikachuNotIncluded": "Festtags-Pikachu nicht enthalten!"
  },
  "newYears": {
    "happyNewYear": "Frohes neues Jahr!"
```

File: src/locales/pt_BR/splash-messages.json [hunk 7]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} Batalhas Ganhas!",
+  "battlesWon": "Batalhas Ganhas!",
   "joinTheDiscord": "Junte-se ao Discord!",
   "infiniteLevels": "Níveis Infinitos!",
-  "everythingStacks": "Tudo Acumula!",
+  "everythingIsStackable": "Everything Is Stackable*!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.json:5]:
```suggestion
  "everythingIsStackable": "Tudo Pode Ser Acumulado*!",
```

File: src/locales/pt_BR/splash-messages.json [hunk 8]
```diff
@@ -14,9 +14,9 @@
   "infiniteFusionAtHome": "Infinite Fusion da Shopee!",
   "brokenEggMoves": "Mov. de Ovo Apelões!",
   "magnificent": "Magnífico!",
-  "mubstitute": "Mubstituto!",
+  "doPeopleReadThis": "As Pessoas Leem Isso?",
   "thatsCrazy": "Que Doidera!",
-  "oranceJuice": "Suco de Laranja!",
+  "gottaCatchEmAll": "Gotta Catch 'Em All!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.json:19]:
```suggestion
  "gottaCatchEmAll": "Temos Que Pegar!",
```

File: src/locales/pt_BR/splash-messages.json [hunk 9]
```diff
@@ -25,12 +25,120 @@
   "moreAddictiveThanIntended": "Mais Viciante do que Planejado!",
   "mostlyConsistentSeeds": "Consistente (na Maioria das Vezes)!",
   "achievementPointsDontDoAnything": "Pontos de Conquista Não Fazem Nada!",
-  "youDoNotStartAtLevel": "Você Não Começa no Nível 2000!",
-  "dontTalkAboutTheManaphyEggIncident": "Não Fale do Incidente do Ovo de Manaphy!",
+  "nothingBeatsAJellyFilledDonut": "Nothing Beats a Jelly-Filled Donut!",
+  "dontTalkAboutTheTinkatonIncident": "Don't Talk About the Tinkaton Incident!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.json:29]:
```suggestion
  "nothingBeatsAJellyFilledDonut": "Nada Melhor Que Um Donut Recheado Com Geleia!",
  "dontTalkAboutTheTinkatonIncident": "Não Fale Sobre o Incidente do Tinkaton!",
```

File: src/locales/pt_BR/splash-messages.json [hunk 10]
```diff
@@ -25,12 +25,120 @@
   "moreAddictiveThanIntended": "Mais Viciante do que Planejado!",
   "mostlyConsistentSeeds": "Consistente (na Maioria das Vezes)!",
   "achievementPointsDontDoAnything": "Pontos de Conquista Não Fazem Nada!",
-  "youDoNotStartAtLevel": "Você Não Começa no Nível 2000!",
-  "dontTalkAboutTheManaphyEggIncident": "Não Fale do Incidente do Ovo de Manaphy!",
+  "nothingBeatsAJellyFilledDonut": "Nada Melhor Que Um Donut Recheado Com Geleia!",
+  "dontTalkAboutTheTinkatonIncident": "Não Fale Sobre o Incidente do Tinkaton!",
   "alsoTryPokengine": "Também Jogue Pokéngine!",
   "alsoTryEmeraldRogue": "Também Jogue Emerald Rogue!",
   "alsoTryRadicalRed": "Também Jogue Radical Red!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.json:32]:
```suggestion
  "alsoTryPokengine": "Jogue Também Pokéngine!",
  "alsoTryEmeraldRogue": "Jogue Também Emerald Rogue!",
  "alsoTryRadicalRed": "Jogue Também Radical Red!",
```

File: src/locales/pt_BR/splash-messages.json [hunk 11]
```diff
@@ -14,9 +14,9 @@
   "infiniteFusionAtHome": "Infinite Fusion da Shopee!",
   "brokenEggMoves": "Mov. de Ovo Apelões!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/splash-messages.json:15]:
```suggestion
  "brokenEggMoves": "Movimentos de Ovo Apelões!",
```

**Review by josericardo-fo — APPROVED** [review]

**Comment by KimJeongSun** [comment]
@Adri1 
Is there any way to know the background or meme from which the updated splash texts come from when translating them?
Some can be easily identified by translators or users from different cultural and language backgrounds, but that's not the case for all.

**Comment by NicusPulcis** [comment]
italian localization done (left a couple of texts in english since they had no reason to be translated and wouldn't be fun otherwise)

File: src/locales/de/splash-messages.json [hunk 12]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} Kämpfe gewonnen!",
+  "battlesWon": "Kämpfe gewonnen!",
```
Comment (CodeTappert) [inline src/locales/de/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} Kämpfe gewonnen!",
```

File: src/locales/es/splash-messages.json [hunk 13]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "¡{{count, number}} Batallas ganadas!",
+  "battlesWon": "¡Batallas ganadas!",
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:2]:
```suggestion
  "battlesWon": "¡{{count, number}} Batallas ganadas!",
```

File: src/locales/fr/splash-messages.json [hunk 14]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} combats gagnés !",
+  "battlesWon": "Combats gagnés !",
```
Comment (CodeTappert) [inline src/locales/fr/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} combats gagnés !",
```

File: src/locales/it/splash-messages.json [hunk 15]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} Battaglie Vinte!",
+  "battlesWon": "Battaglie Vinte!",
```
Comment (CodeTappert) [inline src/locales/it/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} Battaglie Vinte!",
```

File: src/locales/ja/splash-messages.json [hunk 16]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "勝ったバトル：{{count, number}}回！",
+  "battlesWon": "Battles Won!",
```
Comment (CodeTappert) [inline src/locales/ja/splash-messages.json:2]:
```suggestion
  "battlesWon": "勝ったバトル：{{count, number}}回！",
```

File: src/locales/ko/splash-messages.json [hunk 17]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} 전투에서 승리하세요!",
+  "battlesWon": "Battles Won!",
```
Comment (CodeTappert) [inline src/locales/ko/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} 전투에서 승리하세요!",
```

File: src/locales/pt_BR/splash-messages.json [hunk 18]
```diff
@@ -1,22 +1,22 @@
 {
-  "battlesWon": "{{count, number}} Batalhas Ganhas!",
+  "battlesWon": "Batalhas Ganhas!",
```
Comment (CodeTappert) [inline src/locales/pt_BR/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} Batalhas Ganhas!",
```

File: src/locales/zh_CN/splash-messages.json [hunk 19]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} 场胜利！",
+  "battlesWon": "Battles Won!",
```
Comment (CodeTappert) [inline src/locales/zh_CN/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} 场胜利！",
```

File: src/locales/zh_TW/splash-messages.json [hunk 20]
```diff
@@ -1,10 +1,10 @@
 {
-  "battlesWon": "{{count, number}} 勝利場數！",
+  "battlesWon": "Battles Won!",
```
Comment (CodeTappert) [inline src/locales/zh_TW/splash-messages.json:2]:
```suggestion
  "battlesWon": "{{count, number}} 勝利場數！",
```

File: src/locales/es/splash-messages.json [hunk 21]
```diff
@@ -2,9 +2,9 @@
   "battlesWon": "¡{{count, number}} Batallas ganadas!",
   "joinTheDiscord": "¡Únete al Discord!",
   "infiniteLevels": "¡Niveles infinitos!",
-  "everythingStacks": "¡Todo se acumula!",
+  "everythingIsStackable": "Everything Is Stackable*!",
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:5]:
```suggestion
```

File: src/locales/es/splash-messages.json [hunk 22]
```diff
@@ -14,9 +14,9 @@
   "infiniteFusionAtHome": "¡Infinite Fusion en casa!",
   "brokenEggMoves": "¡Movimientos Huevo rotos!",
   "magnificent": "¡Magnífico!",
-  "mubstitute": "¡Mubstituto!",
+  "doPeopleReadThis": "Do People Read This?",
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:17]:
```suggestion
  "doPeopleReadThis": 
```

File: src/locales/es/splash-messages.json [hunk 23]
```diff
@@ -14,9 +14,9 @@
...
   "thatsCrazy": "¡De locos!",
-  "oranceJuice": "¡Zumo de narancia!",
+  "gottaCatchEmAll": "Gotta Catch 'Em All!",
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:19]:
```suggestion
```

File: src/locales/es/splash-messages.json [hunk 24]
```diff
@@ -25,12 +25,120 @@
   "moreAddictiveThanIntended": "¡Más adictivo de lo previsto!",
   "mostlyConsistentSeeds": "¡Semillas CASI consistentes!",
   "achievementPointsDontDoAnything": "¡Los Puntos de Logro no hacen nada!",
-  "youDoNotStartAtLevel": "¡No empiezas al nivel 2000!",
-  "dontTalkAboutTheManaphyEggIncident": "¡No se habla del Incidente Manaphy!",
+  "nothingBeatsAJellyFilledDonut": "Nothing Beats a Jelly-Filled Donut!",
+  "dontTalkAboutTheTinkatonIncident": "Don't Talk About the Tinkaton Incident!",
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:29]:
```suggestion
```

File: src/locales/es/splash-messages.json [hunk 25]
```diff
@@ -25,12 +25,120 @@
...
+  "xmas": {
+    "happyHolidays": "Happy Holidays!",
+    "delibirdSeason": "Delibird Season!",
+    "unaffilicatedWithDelibirdServices": "Unaffiliated With Delibird Services!",
+    "diamondsFromTheSky": "Diamonds From the Sky!",
+    "holidayStylePikachuNotIncluded": "Holiday Style Pikachu Not Included!"
+  },
+  "newYears": {
+    "happyNewYear": "Happy New Year!"
+  }
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:143]:
```suggestion
```

File: src/locales/ja/splash-messages.json [hunk 26]
```diff
@@ -1,36 +1,3 @@
 {
   "battlesWon": "勝ったバトル：{{count, number}}回！",
```
Comment (CodeTappert) [inline src/locales/ja/splash-messages.json:2]:
```suggestion
  "battlesWon": "勝ったバトル：{{count, number}}回！"
```

File: src/locales/es/splash-messages.json [hunk 27]
```diff
@@ -25,12 +22,8 @@
   "moreAddictiveThanIntended": "¡Más adictivo de lo previsto!",
   "mostlyConsistentSeeds": "¡Semillas CASI consistentes!",
   "achievementPointsDontDoAnything": "¡Los Puntos de Logro no hacen nada!",
-  "youDoNotStartAtLevel": "¡No empiezas al nivel 2000!",
-  "dontTalkAboutTheManaphyEggIncident": "¡No se habla del Incidente Manaphy!",
   "alsoTryPokengine": "¡Prueba también Pokéngine!",
   "alsoTryEmeraldRogue": "¡Prueba también Emerald Rogue!",
   "alsoTryRadicalRed": "¡Prueba también Radical Red!",
   "eeveeExpo": "¡Eevee Expo!",
```
Comment (CodeTappert) [inline src/locales/es/splash-messages.json:28]:
```suggestion
  "eeveeExpo": "¡Eevee Expo!"
```

File: src/locales/ko/splash-messages.json [hunk 28]
```diff
@@ -13,24 +12,16 @@
...
-  "dontTalkAboutTheManaphyEggIncident": "마나피 알 사건에 대해선 이야기하지 맙시다!",
   "alsoTryPokengine": "Pokéngine도 해봐요!",
   "alsoTryEmeraldRogue": "Emerald Rogue도 해봐요!",
   "alsoTryRadicalRed": "Radical Red도 해봐요!",
-  "eeveeExpo": "Eevee Expo!",
-  "ynoproject": "YNOproject!",
-  "breedersInSpace": "충격!우주에 브리더 진짜 계심ㄷㄷ"
-}
\ No newline at end of file
+  "breedersInSpace": "충격!우주에 브리더 진짜 계심ㄷㄷ",
```
Comment (CodeTappert) [inline src/locales/ko/splash-messages.json:26]:
```suggestion
  "breedersInSpace": "충격!우주에 브리더 진짜 계심ㄷㄷ"
```

File: src/locales/zh_CN/splash-messages.json [hunk 29]
```diff
@@ -25,12 +22,9 @@
...
   "achievementPointsDontDoAnything": "成就点数没有任何用处！",
-  "youDoNotStartAtLevel": "你不能第一关就改个2000级！",
-  "dontTalkAboutTheManaphyEggIncident": "别再提玛纳霏蛋事件了！",
   "alsoTryPokengine": "也玩玩看Pokéngine！",
   "alsoTryEmeraldRogue": "也玩玩看绿宝石肉鸽！",
   "alsoTryRadicalRed": "也玩玩看激进红！",
   "eeveeExpo": "伊布博览会！",
-  "ynoproject": "Yume Nikki 页游项目！",
-  "breedersInSpace": "饲养员也能上太空？"
+  "breedersInSpace": "饲养员也能上太空？",
```
Comment (CodeTappert) [inline src/locales/zh_CN/splash-messages.json:29]:
```suggestion
  "breedersInSpace": "饲养员也能上太空？"
```

**Review by CodeTappert — APPROVED** [review]

**Review by damocleas — APPROVED** [review]

**Pull request merged**